src/cmds/qstat.c

```diff
@@ -545,6 +548,32 @@ prt_attr(char *name, char *resource, char *value, int one_line) {
 #define SIZETSK 5	/* length of tsk in printf */
 #define SIZEUSER 15	/* length of user name in printf */
 
+
+/**
+ * @brief
+ *	Check if value is too long, truncate and append with * if needed
+ *
+ * @param[in] value - value that may be truncated
+ * @param[in] len - non-wide length that must be met
+ * @param[in] wide_len - wide length that must be met
+ * @param[in] wide - whether or not wide format is used
+ *
+ */
+
+static void
+trunc_value(char *value, int len, int wide_len, int wide)
```

Comment:
C99 allows you to use bool for the wide parameter. Why do you need two length parameters if you'll only ever use one or the other? Can we remove wide_len?

Reply:
If I change `trunc_value` to only take one length, then everywhere I call it I have to put `if (wide) ... else` blocks.
I kept wide as an int to follow the style of the file, but I can change it if you want.

Reply:
Ahh... thanks for explaining, that makes sense.